postgres/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM postgres:latest
+COPY postgres/setup.sh /docker-entrypoint-initdb.d/setup.sh
+COPY postgres/standby_setup.sh /standby_setup.sh
+RUN chmod 0666 /docker-entrypoint-initdb.d/setup.sh
+RUN chmod 0700 /standby_setup.sh
```

Comment:
nitpick: you should move the chmod for setup.sh to before the copy for standby_setup.sh, so that it doesn't need to be needlessly repeated if you're doing a local rebuild and only standby_setup.sh has changed.